src/libclipper/test/container_test.cpp

```diff
@@ -0,0 +1,43 @@
+#include <gtest/gtest.h>
+
+#include <clipper/containers.hpp>
+
+using namespace clipper;
+
+namespace {
+
+TEST(ContainerTests,
+     ModelContainerAverageThroughputUpdatesCorrectlyFewSamples) {
+  VersionedModelId model("test", 1);
+  ModelContainer container(model, 0, InputType::Doubles);
+  std::array<long, 4> single_task_latencies_micros = {500, 2000, 3000, 5000};
```

Comment:
The initialization is less verbose compared to initializing a vector and adding the elements one-by-one.